scripts/README.md

```diff
@@ -1,32 +1,39 @@
-# Usage
+# Adding/updating translations
+To update the translations for a given locale, use
+[`scripts/add_translation.py`] (https://github.com/CenterForOpenScience/isp/blob/develop/scripts/add_translation.py)
 
-## Adding/ updating consent form text
-To update consent form text, see [`scripts/consent_form_json.py`](https://github.com/CenterForOpenScience/isp/blob/develop/scripts/consent_form_json.py)
+1. Run the command `python add_translation.py`, this is the main script to be used. `add_translation.py` uses both [`scripts/format_translations.py`](https://github.com/CenterForOpenScience/isp/blob/develop/scripts/format_translations.py) and [`scripts/consent_form_json.py`](https://github.com/CenterForOpenScience/isp/blob/develop/scripts/consent_form_json.py).
 
-You may need to update the `FOLDER_ID` in the script to the particular Google Drive folder ID with consent 
-forms for your desired language.
+2. `add_translation.py` will first download the CSV file needed for the translation from the google drive folder. Currently, each language has its own google drive folder. The ISP project main folder can be located [`here`](https://drive.google.com/drive/u/0/folders/0BxGwKGgJtw4WYVpJVllsMk84Wms). All language translation folders are stored under `/Samples/Translations/`.
```

Comment:
Overall these instructions are an improvement over what I had before; very good!

One small tweak requested here, though: The user will be asked for a folder ID when they run the script. We should tell them how to figure out the folder ID for their translation (and how to know *which* folder to give the script at all, eg the consent form folder, the parent folder for that language, the overall translations folder... etc).